src/index.vue

```diff
@@ -31,7 +31,9 @@
         v-if="(multiple && taggable && modelValue.length === 0) || (searchable === false && taggable === false)"
       >
         <div class="vue-input">
-          <input :placeholder="innerPlaceholder" readonly @click="focus" />
+          <slot name="label" :label="innerPlaceholder" :selected="selected">
```

Comment:
Since users may want to customize the placeholder too, we could only expose `selected`.